### PR TITLE
fix(core):13708 layersInAreaAndEventLayerResource metric is sent only after refresh the page

### PR DESCRIPTION
**Pull request by vkozio**
https://kontur.fibery.io/Tasks/Task/layersInAreaAndEventLayerResource-metric-is-sent-only-after-refresh-the-page-13708

metrics is stopped when page is hidden

additional improvements:
removed currentApplicationAtom and it usages (we have appId in appConfig)
added more missing names for atoms

### CODE REVIEW

File: src/core/app/init.ts [hunk 1]
```diff
@@ -41,9 +40,6 @@ export async function appInit() {
 
   updateAppConfig(appInfo);
 
-  // FIXME: refactor remove currentApplicationAtom dependency in other atoms
-  currentApplicationAtom.set.dispatch(initialState.app);
```
Comment (propakov) [inline src/core/app/init.ts:45]: currentApplicationAtom still stays in src/features/create_layer/readme.md
need to remove from docs as it doesn't exist

File: src/features/create_layer/atoms/editableLayersListResource.ts [hunk 2]
```diff
@@ -1,42 +1,15 @@
...
-);
-
 export const editableLayersListResource = createAsyncAtom(
-  editableLayersListDependencyAtom,
-  async (params, abortController) => {
-    if (!params?.appId) return null;
-
+  null,
+  async (_, abortController) => {
     const responseData = await apiClient.post<EditableLayers[]>(
```
Comment (propakov) [inline src/features/create_layer/atoms/editableLayersListResource.ts:10]: should we also add it here? `if (!appConfig.id) return null;`

Comment (vkozio) [reply src/features/create_layer/atoms/editableLayersListResource.ts:10]: appConfig.id cannot be empty

File: src/core/app/init.ts (hunk 1 above)
Comment (vkozio) [reply src/core/app/init.ts:45]: good point

**Review by propakov — APPROVED** [review]

File: src/features/create_layer/atoms/editableLayersListResource.ts (hunk 2 above)
Comment (propakov) [reply src/features/create_layer/atoms/editableLayersListResource.ts:10]: so why to handle it here? 
https://github.com/konturio/disaster-ninja-fe/pull/402/files#diff-f926c1c66cd1d38ab061e5691b47a9059b0831317f6a5fe7ab9a7da93e8c9ac4R10

Comment (vkozio) [reply src/features/create_layer/atoms/editableLayersListResource.ts:10]: fixed

**Pull request merged**